controllers/errors.js

```diff
@@ -0,0 +1,11 @@
+exports.handle404s = (req, res, next) => {
+  res.status(404).send({ msg: "Cannot find what you wanted" });
+};
+
+exports.handle500s = (err, req, res, next) => {
+  if (err.status === 500) {
```

Comment:
in the 500 error handler you don't need to check for the status - the idea with this handler is that if the error makes it here then it is automatically a 500 because it is not being handled elsewhere